trading_calendars/exchange_calendar_xpra.py

```diff
@@ -0,0 +1,164 @@
+#
+# Copyright 2019 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import time
+import pandas as pd
+from pandas.tseries.holiday import (
+    Easter,
+    EasterMonday,
+    Holiday,
+)
+from pandas.tseries.offsets import Day
+from pytz import UTC, timezone
+
+from .common_holidays import (
+    new_years_day,
+    new_years_eve,
+    european_labour_day,
+    christmas,
+    christmas_eve,
+)
+from .trading_calendar import (
+    TradingCalendar,
+    HolidayCalendar,
+)
+
+
+NewYearsDay = new_years_day()
+
+GoodFriday = Holiday(
+    "Good Friday",
+    month=1,
+    day=1,
+    offset=[Easter(), Day(-2)],
+    start_date='2013'
```

Comment:
This might be worth a quick comment that we can't use the built-in `GoodFriday` since we need a start date.

Reply:
Agreed, will do